src/style.css

```diff
@@ -28,23 +28,56 @@
   height: 100%;
   width: 100%;
   touch-action: none;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
 }
 .playkit-floating-container.playkit-floating-active {
   z-index: 9999;
   position: fixed;
+  border-radius: 8px;
 }
 .playkit-floating-container.playkit-floating-active.draggable {
   cursor: grab;
 }
 
+#playkit-floating-dismissible-container {
+  display: none;
+  align-items: center;
+  flex-direction: row;
+  width: 100%;
+  height: 32px;
+  background: #242424;
+  border-radius: 8px 8px 0 0;
+  .playkit-dismissible-text {
+    color: white;
+    display: flex;
+    width: 100%;
+    margin-left: 8px;
+    font-size: 14px;
+    font-weight: 700;
+    font-family: Lato, Helvetica Neue, Segoe UI, sans-serif;
```

Comment:
as far as i remember we don't define font-family in the player css ourselves but... let the host page define it i think ? i remember someone looked at this before, wasn't it you ?